src/utils.js

```diff
@@ -0,0 +1,21 @@
+export function throttle (callback, wait, immediate = false) {
```

Comment:
I see the project already uses `lodash` & `lodash` has `throttle` [method](https://lodash.com/docs#throttle). Can you add a comment why we need a custom one?

Reply:
I used it but i did not see the previous call being cancelled so i did not want to fight another framework.

Reply:
I'm not sure what you mean by previous calls being cancelled @karimkawambwa... If you want to cancel I think you have to do that manually, `_.throttle` just ensures the calls are spaced out in time. If you want the other calls made within a given time to automatically be dropped, I think `_.debounce` is what you need.

Reply:
Cool i want `_.debounce ` behavior